### PR TITLE
Fix to line-coverage job failure

### PR DESCRIPTION
**Pull request by rakshithakamath94**
Hi @deepshikhaaa @mscherer ,

Please review this PR.

Regards,
Rakshitha Kamath

### CODE REVIEW

File: build-gluster-org/jobs/lcov.yml [hunk 1]
```diff
@@ -28,14 +28,14 @@
...
           script: |
-              sudo reboot
+              sudo chown -R jenkins:jenkins $WORKSPACE
     - post-tasks:
         - matches:
-            - log-text: Building remotely
+            - log-text: Build was aborted
           script: |
-              sudo chown -R jenkins:jenkins $WORKSPACE
+              sudo reboot
```
Comment (deepshikhaaa) [inline build-gluster-org/jobs/lcov.yml:38]: Can you call out this `scripts/regression-abort.sh` instead.

Comment (rakshithakamath94) [reply build-gluster-org/jobs/lcov.yml]: The script regression-abort.sh includes steps to archive logs and upload it to logs server, should we do that with line-coverage?

Comment (deepshikhaaa) [reply build-gluster-org/jobs/lcov.yml:38]: Yes! We should upload the logs as well.

Comment (rakshithakamath94) [reply build-gluster-org/jobs/lcov.yml:38]: Oh! Okay.

Comment (rakshithakamath94) [reply build-gluster-org/jobs/lcov.yml:38]: Okay.

**Pull request merged**